stdlib/public/core/Collection.swift

```diff
@@ -507,29 +506,26 @@ public struct IndexingIterator<
 /// collection can contain zero or more of the original collection's elements
 /// and shares the original collection's semantics.
 ///
-/// The following example, creates a `firstWord` constant by using the
-/// `prefix(_:)` method to get a slice of the `text` string's `characters`
-/// view.
+/// The following example creates a `firstWord` constant by using the
+/// `prefix(_:)` method to get a slice of the `text` string.
 ///
-///     let firstWord = text.characters.prefix(7)
-///     print(String(firstWord))
+///     let firstWord = text.prefix(7)
+///     print(firstWord)
 ///     // Prints "Buffalo"
 ///
 /// You can retrieve the same slice using other methods, such as finding the
-/// terminating index, and then using the `prefix(upTo:)` method, which takes
-/// an index as its parameter, or by using the view's ranged subscript.
+/// terminating index, and the using the string's ranged subscript, or by
+/// using the `prefix(upTo:)` method, which takes an index as its parameter.
```

Comment:
I think you should drop mention of that; IMO we'll probably obsolete (or at least deprecate) it for Swift 4